### PR TITLE
Bug 1144593 - Recalculate statusbar appearance after closing rocketbar r=gmarty

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by albertopq** [comment]
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=948db10042c265173471e52a298c28d2b8844ab2

File: apps/system/test/unit/statusbar_test.js [hunk 1]
```diff
@@ -797,6 +797,16 @@ suite('system/Statusbar', function() {
       assert.isTrue(setAppearanceStub.calledWith(app));
     });
 
+    test('rocketbar-deactivated', function() {
+      StatusBar.element.classList.add('hidden');
+      assert.isTrue(StatusBar.element.classList.contains('hidden'));
```
Comment (gmarty) [inline apps/system/test/unit/statusbar_test.js]: Not sure I understand the purpose of this line.

Comment (albertopq) [inline apps/system/test/unit/statusbar_test.js]: To be honest, just copied and pasted the test for 'stackchanged' event, as we should do the same... but you are right, it doesn't make any sense. I'll remove both in order to clean it up. Thanks!

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=05634a3456a7e104bb4ddac7b9ed04e7a9d1b837

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=36108300872944e0627fd69e09236ce87b48d429

**Pull request merged**